docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,318 @@
+- Use `brigade.js` for the `Upload a default brigade.js script` option
+- Feel free to select a `secret for the Generic Gateway` (we're using `mysecret` in the example below)
+- You can leave all other options at their default values (just press Enter on every interactive prompt)
+
+```
+? Project Name brigadecore/empty-testbed
+? Full repository name github.com/brigadecore/empty-testbed
+? Clone URL (https://github.com/your/repo.git) https://github.com/brigadecore/empty-testbed.git
+? Add secrets? No
+Auto-generated a Shared Secret: "VdNQ1eO4qVBXNmZZgNcO51CP"
+? Configure GitHub Access? No
+? Configure advanced options Yes
+? Custom VCS sidecar (enter 'NONE' for no sidecar) NONE
+? Build storage size
+? SecretKeyRef usage No
+? Build storage class default
+? Job cache storage class default
+? Worker image registry or DockerHub org
+? Worker image name
+? Custom worker image tag
+? Worker image pull policy IfNotPresent
+? Worker command yarn -s start
+? Initialize Git submodules No
+? Allow host mounts No
+? Allow privileged jobs No
+? Image pull secrets
+? Default script ConfigMap name
+? brigade.js file path relative to the repository root
+? Upload a default brigade.js script brigade.js
+? Secret for the Generic Gateway (alphanumeric characters only). Press Enter if you want it to be auto-generated mysecret
+Project ID: brigade-4897c99315be5d2a2403ea33bdcb24f8116dc69613d5917d879d5f
+```
+
+*Yes, we do know that the process to create a non-VCS/Generic Gateway Project is a bit cumbersome. This is going to be fixed in [#816](https://github.com/brigadecore/brigade/issues/816)*
+
+### Send a SimpleEvent to the Generic Gateway
+
+Next, we should send a simple JSON object to test our Generic Gateway. By default, Generic Gateway is not exposed to the outside world, since it is created as a [Cluster IP Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+To send traffic to a Cluster IP Service, we can use `kubectl port-forward`. The following command will open a tunnel from local port 8081 to port 8081 on the Generic Gateway Service.
+
+```bash
+kubectl port-forward service/brigade-brigade-generic-gateway 8081:8081
+```
+
+To send a JSON message to Generic Gateway, open a new shell and try this curl command:
+
+```bash
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{
+    "key1": "value1",
+    "key2": "value2"
+}' \
+  http://localhost:8081/simpleevents/v1/brigade-4897c99315be5d2a2403ea33bdcb24f8116dc69613d5917d879d5f/mysecret
+```
+
+Pay attention to the Generic Gateway URL for a SimpleEvent, it is `HOST:PORT/simpleevents/v1/Project-ID/genericGatewaySecret`
+
+If all is well, you'll get back a 200 result: 
+
+```json
+{"status":"Success. Build created"}
+```
+
+### View this Build's log
+
+Let's now see what happened in our Worker Pods:
+
+```bash
+kubectl get pods
+```
+
+You'll see that two Pods were created for this Build. The worker Pod and the `echosimpleevent` Pod.
```

Comment:
Could you please link to how Brigade works here?

Reply:
You mean this document, right? https://docs.brigade.sh/topics/design/ Adding